src/metrics_server/connection_metrics.ts

```diff
@@ -95,6 +97,7 @@ export function isValidConnectionMetricsReport(
 
   const MIN_BYTES_TRANSFERRED = 0;
   const MAX_BYTES_TRANSFERRED = 1 * Math.pow(2, 40); // 1 TB.
+  const MIN_TUNNEL_TIME = 0;
   for (const userReport of testObject.userReports) {
     // We require at least the userId or the country to be set.
     if (!userReport.userId && (userReport.countries?.length ?? 0) === 0) {
```

Comment:
Let's drop userId. We don't use it. We need to make sure the collector doesn't break though.

With Tunnel Time, we should consider dropping serverId as well.

Reply:
I'll add a TODO and look at in a separate PR.

Reply:
See #1528.